liquidhaskell-boot/src/Language/Haskell/Liquid/Types/Specs.hs

```diff
@@ -391,6 +393,7 @@ data Spec ty bndr  = Spec
   , impSigs    :: ![(F.Symbol, F.Sort)]                               -- ^ Imported variables types
   , expSigs    :: ![(F.Symbol, F.Sort)]                               -- ^ Exported variables types
   , asmSigs    :: ![(F.LocSymbol, ty)]                                -- ^ Assumed (unchecked) types; including reflected signatures
+  , asmReflectSigs :: ![(F.LocSymbol, F.LocSymbol)]                   -- ^ Assumed (unchecked) types; including reflected signatures
```

Comment:
The comment here is not very telling. Maybe mention the roles that each component of the pair plays.